### PR TITLE
Task 05 Modules and Services

### PR DESCRIPTION
**Pull request by ShevtsovAnton**

### CODE REVIEW

File: src/app/features/courses/services/courses.service.ts [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+  remove(courseToRemove: CoursesItemModel): void {
+    this.coursesList.map( (course, index) => {
+      if (course.id === courseToRemove.id) {
+        this.coursesList = [
+          ...this.coursesList.slice(0, index),
+          ...this.coursesList.slice(index + 1)
+        ];
+      }
+    });
```
Comment (vnovikov-clgx) [inline src/app/features/courses/services/courses.service.ts]: minor: 
this.coursesList = this.coursesList.reduce((allCurses, course) => {
if(course.id !== courseToRemove.id) {
allCurses.push(course);
}
return allCurses;
}, [] )

File: src/app/features/login/components/login/login.component.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<div class="login__container">
+  <h2 class="login__header">Login</h2>
+  <form (submit)="onSubmit(username, password)" class="login__form">
```
Comment (vnovikov-clgx) [inline src/app/features/login/components/login/login.component.html]: Get this `username, password` inside of onSubmit function, don't pass it as a param

File: src/app/features/login/components/login/login.component.scss [hunk 3]
```diff
@@ -0,0 +1,44 @@
+@import '../../../../../styles/variables.scss';
+
+.login {
+    &__container {
+        width: 400px;
+        border: 1px solid black;
+        border-radius: 5px;
+        margin: auto;
+        background: white;
```
Comment (vnovikov-clgx) [inline src/app/features/login/components/login/login.component.scss]: minor: use variables

File: src/app/features/login/components/login/login.component.scss [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    }
+
+    &__header {
+        margin: 0px;
+        padding: 5px 10px;
+        background-color: rgb(211, 208, 208);
```
Comment (vnovikov-clgx) [inline src/app/features/login/components/login/login.component.scss]: minor: use variables

File: src/app/features/courses/services/courses.service.ts (hunk 1 above)
Comment (ShevtsovAnton) [reply src/app/features/courses/services/courses.service.ts]: cool, thanks for the advice.

File: src/app/features/login/components/login/login.component.scss (hunk 3 above)
Comment (ShevtsovAnton) [reply src/app/features/login/components/login/login.component.scss]: moved to a variable

File: src/app/features/login/components/login/login.component.scss (hunk 4 above)
Comment (ShevtsovAnton) [reply src/app/features/login/components/login/login.component.scss]: moved to a variable

**Pull request merged**